website/content/docs/connect/config-entries/jwt-provider.mdx

```diff
@@ -952,6 +952,22 @@ Defines behavior for caching the validation result of previously encountered JWT
 </Tab>
 </Tabs>
 
+## Metrics
+
+The following `envoy` metrics can be used to track jwt authentication details. 
+
+~> **Note:** Envoy does not currently provide any documentation on these metrics.
+
+```yaml
+http.ingress_http.jwt_authn.allowed
```

Comment:
Do you know if this will this always be `ingress_http`? 🤔 Is maybe the name of the http filter?

Reply:
good catch, i was going by the rfc but let me confirm this locally and in the envoy too